packages/portfolio-contract/package.json

```diff
@@ -33,9 +33,11 @@
     "ts-blank-space": "^0.6.1"
   },
   "dependencies": {
+    "@aglocal/fast-usdc-deploy": "^0.1.0",
```

Comment:
ymax/portfolio shouldn't depend on fast-usdc. especially: no contract package should depend on a deploy package